src/logging/writers/ascii/Ascii.cc

```diff
@@ -255,52 +255,52 @@ bool Ascii::InitFilterOptions() {
     const WriterInfo& info = Info();
 
     // Set per-filter configuration options.
-    for ( WriterInfo::config_map::const_iterator i = info.config.begin(); i != info.config.end(); ++i ) {
-        if ( strcmp(i->first, "tsv") == 0 ) {
-            if ( strcmp(i->second, "T") == 0 )
+    for ( const auto& [key, value] : info.config ) {
```

Comment:
I think the bigger bang for the buck would be to switch `info.config` to std::map<std::string, std::string>` at some point to ditch all the `strcmp` calls :-/